source/get-started/bare-metal-install-desktop.rst

```diff
@@ -344,8 +344,10 @@ boot partition
 
 #. Select :guilabel:`Add`.
 
-swap partition
---------------
+swap partition (optional)
+-------------------------
+
+A swapfile is generated by default during installation. However, if you wish to manually create a swap partition, follow the steps below. 
```

Comment:
```suggestion
A swapfile is generated by default during installation. However, if you instead wish to create a swap partition, follow the steps below. 
```

Reply:
@mdhorn how about?

A swapfile is generated by default during installation. However, if you instead wish to create a swap partition, check the <Advanced Installation swap link>?

Reply:
So is the word "manually" what's at issue? If so, how about a compromise?:

A swapfile is generated by default during installation. However, if you **prefer** to create a swap partition, follow the steps below. 

"Instead" is already implied by "However, if you prefer..."